test/hull_test.cpp

```diff
@@ -197,4 +197,29 @@ TEST(Hull, FailingTest2) {
   }
 #endif
   EXPECT_TRUE(isMeshConvex(hull, 2.13966e-05));
+}
+
+TEST(Hull, DisabledFaceTest) {
```

Comment:
When you say it has a disabled face, do you mean during the coarse of the algorithm it disables a face? Did you verify this test fails if you revert whatever the fix was?

Reply:
Yes, and that disabled face is linked to one of the `halfEdge` returned after the `CreateConvexHalfedgeMesh` function call. The algorithm doesn't remove them during computation to reduce time. But, we need to filter those at the end.

Yes, I verified that the test fails without the fix.